### ISSUE TITLE
how dispatch job on different executor

### ISSUE THREAD

**Issue by xujiapei** [azkaban/azkaban#1864]
Hi guys:
I am new on Azkaban. Recently, I builded Azkaban 3.x . I tried to run the command type on Multiple Executor Mode. But I find that
the dispatch level is flow , not the job's level .So I want to know if the Azkaban 3.x can dispatch on the job's level.
e,g: My project has one flow ,and this flow has some jobs.

**Comment by juhoautio** [comment]
No, it's not supported. Job-level execution is entirely managed by a single executor instance.

As a work-around you could split your jobs into multiple flows (grouped by executor id to use) and use the rather new conditional triggering feature to start each flow when its flow dependencies have finished.

**Comment by xujiapei** [comment]
thx

**Comment by happyapple668** [comment]
pls close the issue
